src/components/gallery/cards/assetCard.vue

```diff
@@ -21,29 +21,30 @@ export default defineComponent({
 <template lang="pug">
 q-card
   q-card-section
+    q-badge.text-subtitle2.float-right(
+      rounded,
+      color='primary',
+      label=' Silver'
```

Comment:
This colour should actually change based on the tier. So some logic needed. Just make a computed property

Reply:
The colour of the badges should actually change according to the tier. If there is no tier. Don't show the badge